charts/substra-backend/templates/deployment-server.yaml

```diff
@@ -27,89 +27,119 @@ spec:
-              value: {{ .Values.organization.name }}
-            - name: BACKEND_DB_NAME
-              value: {{ .Values.postgresql.postgresqlDatabase }}
-            - name: BACKEND_DB_USER
-              value: {{ .Values.postgresql.postgresqlUsername }}
-            - name: BACKEND_DB_PWD
-              value: {{ .Values.postgresql.postgresqlPassword }}
-            - name: DATABASE_HOST
-              value: {{ .Release.Name }}-postgresql
-            - name: DEFAULT_DOMAIN
-              value: "{{ .Values.backend.defaultDomain }}"
-            - name: CELERY_BROKER_URL
-              value: "amqp://{{ .Values.rabbitmq.rabbitmq.username }}:{{ .Values.rabbitmq.rabbitmq.password }}@{{ .Release.Name }}-{{ .Values.rabbitmq.host }}:{{ .Values.rabbitmq.port }}//"
-            - name: BACKEND_DEFAULT_PORT
-              value: {{ .Values.backend.service.port | quote}}
-            - name: BACKEND_PEER_PORT
-              value: "internal"
-            - name: LEDGER_CONFIG_FILE
-              value: /conf/{{ .Values.organization.name }}/substra-backend/conf.json
-            - name: MEDIA_ROOT
-              value: {{ .Values.persistence.hostPath }}/medias/
-            - name: PYTHONUNBUFFERED
-              value: "1"
-          ports:
-            - name: http
-              containerPort: {{ .Values.backend.service.port }}
-              protocol: TCP
-          volumeMounts:
-            - name: data
-              mountPath: {{ .Values.persistence.hostPath }}
-            - name: config
-              mountPath: /conf/{{ .Values.organization.name }}/substra-backend
-              readOnly: true
-            - name: user-cert
-              mountPath: /var/hyperledger/msp/signcerts
-            - name: user-key
-              mountPath: /var/hyperledger/msp/keystore
-            - name: peer-tls-server
-              mountPath: /var/hyperledger/tls/server/pair
-            - name: peer-tls-client
-              mountPath: /var/hyperledger/tls/client/pair
-            - name: orderer-tls-ca
-              mountPath: /var/hyperledger/tls/ord/cert
-            - name: ca
-              mountPath: /var/hyperledger/ca
-          livenessProbe:
-            httpGet:
-              path: /liveness
-              port: http
-              httpHeaders:
-                - name: Accept
-                  value: "text/html;version=0.0, */*;version=0.0"
-            initialDelaySeconds: 60
-            timeoutSeconds: 5
-            periodSeconds: 45
-            failureThreshold: 6
-          readinessProbe:
-            httpGet:
-              path: /readiness
-              port: http
-              httpHeaders:
-                - name: Accept
-                  value: "text/html;version=0.0, */*;version=0.0"
-            initialDelaySeconds: 5
-            timeoutSeconds: 2
-            periodSeconds: 30
-          resources:
-            {{- toYaml .Values.backend.resources | nindent 12 }}
+      - name: {{ template "substra.name" . }}
```

Comment:
is the change in indentation intentional?

Reply:
yes. I followed examples on the official documentation